LogisticApp/ViewModel/Forms/OrderFormViewModel.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using LogisticApp.ViewModel.BaseClass;
+using LogisticApp.Model;
+using System.ComponentModel;
+
+namespace LogisticApp.ViewModel.Forms
+{
+    class OrderFormViewModel : BaseViewModel
+    {
+        public BaseViewModel orderFormViewModel = new OrderFormViewModel();
```

Comment:
There is no reason to keep it's own instance in ViewModel, This works in MainViewModel because MainViewModel controls which user control is being displayed in the window, it keeps instances of these control ViewModels.